app-editors/vis/vis-0.6.ebuild

```diff
@@ -12,17 +15,17 @@ SRC_URI="https://github.com/martanne/vis/releases/download/v${PV}/${P}.tar.gz
 LICENSE="ISC"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="+ncurses selinux test tre"
+IUSE="+ncurses +lua +lpeg selinux test tre"
 RESTRICT="!test? ( test )"
 
 # - Known to also work with NetBSD curses
-# - ::lua package done for using >=dev-lang/lua-5.2
-# which is needed for syntax highlighting and settings but masked in ::gentoo
 DEPEND="dev-libs/libtermkey
 	ncurses? ( sys-libs/ncurses:0= )
+	lua? ( ${LUA_DEPS} )
 	tre? ( dev-libs/tre:= )"
 RDEPEND="${DEPEND}
-	app-eselect/eselect-vi"
+	app-eselect/eselect-vi
+	lua? ( lpeg? ( $(lua_gen_cond_dep '>=dev-lua/lpeg-0.12[${LUA_USEDEP}]') ) )"
```

Comment:
I'm sorry if I missed this the first time, but how is `lpeg` used in this ebuild? Now that I can see, it doesn't have other function that to control optional runtime dependency which is prohibited in Gentoo. You should use `optfeature` instead, which also takes away this `lua_gen_cond_dep` construct...
https://projects.gentoo.org/qa/policy-guide/dependencies.html#optional-runtime-dependencies